src/routes/health/health.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { handler as healthCheck, Health, setRouteHealth } from '.';
+import { Request, Response } from 'express';
+
+describe('API Routing Health Check', () => {
+  let req: Partial<Request> ;
+  let res: Partial<Response>;
+  let status: Health;
+
+  beforeAll(() => {
+    res = {
+      send: jest.fn(),
+      status: jest.fn()
+    };
+    req = {};
+  });
+
+  it('sets route health', () => {
+    status = Health.busy;
+    req.path = '/init';
+    setRouteHealth(status, <Request>req);
```

Comment:
We're not actually checking that anything happens here. We'll need to actually confirm that this function performed the correct task.

Reply:
This has been resolved in the latest changes.